virttest/libvirtd_decorator.py

```diff
@@ -0,0 +1,70 @@
+"""
+A decorator utility functions to apply libvirtd functions.
+
+:copyright: 2008-2009 Red Hat Inc.
+"""
+
+import logging
+import os
+from virttest import data_dir
+from virttest import libvirt_version
+
+
+def get_libvirtd_split_enable_bit():
+    base_cfg_path = os.path.join(data_dir.get_shared_dir(), 'cfg', 'base.cfg')
```

Comment:
What is the reason that we do not read the param in base.cfg directly and simply?
@chunfuwen

Reply:
utils files under avocado-vt can not get params directly, it only get from passing in from runtest (test , params,xx) of tp-libvirt.